avocado/core/job.py

```diff
@@ -389,6 +406,12 @@ def test_parameters(self):
                                                         [])}
         return self._test_parameters
 
+    @property
+    def test_suites(self):
```

Comment:
Before final inclusion, given that this is a public Job API (I'd say), we should work on its docstring.

BTW, the commit gives a strange message IMO because of the `get a list of TestSuite`.  It seems like this method will fetch (get) this data itself, and not return it.  Just noting that because it may be the foundation for the docstring.  Suggestion: "This (property) contains a list of TestSuite objects..."

Reply:
Thanks, Fixed.